python/format/src/graphs.py

```diff
@@ -11,6 +14,94 @@
 Edges = [Any, Any, Any]
 
 
+@dataclasses.dataclass(frozen=True)
+class Node:
```

Comment:
Might be useful to have a small docstring and give a few examples of nodes.

Reply:
Done.